apps/api_web/lib/api_web/views/live_facility_view.ex

```diff
@@ -14,7 +14,7 @@ defmodule ApiWeb.LiveFacilityView do
 
   attributes([:properties, :updated_at])
 
-  def type(_, _), do: "live-facility"
+  def type(_, _), do: "live_facility"
```

Comment:
This needs to stay `live-facility` for the current version. I believe the second argument here is a `conn`.

Reply:
Fixed